app/views/eventRegistrations.html

```diff
@@ -42,20 +42,46 @@ <h2 class="page-title">Registrations</h2>
             <i class="fa fa-chevron-down" ng-show="!showMoreFilters"></i>
             <i class="fa fa-chevron-up" ng-show="showMoreFilters"></i>
           </button>
-          <button class="btn btn-primary" ng-click="refreshRegistrations()"><i class="fa fa-refresh"></i> Refresh</button>
-          <button type="button" class="btn btn-primary dropdown-toggle" dropdown-toggle>
-            Columns
-            <i class="fa fa-caret-down"></i>
+          <button class="btn btn-primary" ng-click="refreshRegistrations()">
+            <i class="fa fa-refresh"></i>
+            Refresh
           </button>
-          <ul class="dropdown-menu dropdown-menu-right reg-views-dropdown">
-            <li ng-repeat="block in blocks" ng-if="block.profileType !== 'NAME' && block.profileType !== 'EMAIL'">
-              <a href="" ng-click="toggleColumn($index)">
-                <i class="fa fa-check-square" ng-show="block.visible"></i>
-                <i class="fa fa-square-o" ng-show="!block.visible"></i>
-                {{block.exportFieldTitle || block.title}}
-              </a>
-            </li>
-          </ul>
+          <div class="btn-group" role="group" dropdown auto-close="outsideClick">
+            <button type="button" class="btn btn-primary dropdown-toggle" dropdown-toggle>
+              Display Columns
```

Comment:
These buttons don't fit on the same line if the screen size is between 768px and 848px:
<img width="829" alt="screen shot 2016-07-12 at 11 41 03 pm" src="https://cloud.githubusercontent.com/assets/756501/16794035/321d9c5e-488a-11e6-8887-8e7bae96c92b.png">

I like the better description. Idk what a good solution is.